config/config.go

```diff
@@ -679,6 +682,19 @@ type SyncFromHost struct {
 	Secrets EnableSwitchWithResourcesMappings `json:"secrets,omitempty"`
 }
 
+type SyncToHostNamespaces struct {
+	Enabled bool `json:"enabled,omitempty" jsonschema:"required"`
```

Comment:
Add a comment above please as we want every field described

Reply:
added 👍